internal/fleet/agent_policy/resource_test.go

```diff
@@ -143,6 +143,19 @@ func TestAccResourceAgentPolicy(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"skip_destroy"},
 			},
+			{
+				SkipFunc: versionutils.CheckIfVersionIsUnsupported(agent_policy.MinVersionInactivityTimeout),
+				Config:   testAccResourceAgentPolicyCreateWithInactivityTimeout(policyName, false, "2m"),
```

Comment:
```suggestion
				Config:   testAccResourceAgentPolicyCreateWithInactivityTimeout(policyName),
```

Lets remove these hard coded parameters from this and the other config helpers where only a single value is used for the parameter.